scripts/set-env.ts

```diff
@@ -1,25 +1,102 @@
 import fs from 'fs';
 import { item, FieldAssignment } from '@1password/op-js';
+import { OnePasswordConnect, FullItem } from '@1password/connect';
+import { FullItemAllOfFields } from '@1password/connect/dist/model/fullItemAllOfFields';
+import config from '../src/config';
 
-export default function createItemFromFile(
+async function createItemWithConnect(
     envFile: string,
     vault: string,
-    title: string
+    title: string,
+    name: string
+): Promise<void> {
+    // Create new connector with HTTP Pooling
+    const op = OnePasswordConnect({
+        serverURL: config.CONNECT_URL,
+        token: config.CONNECT_TOKEN,
+        keepAlive: true,
+    });
+    const opVault = await op.getVault(vault);
+    if (!opVault) throw new Error('Vault not found');
+
+    try {
+        fs.readFile(envFile, 'utf8', async (err, data) => {
+            if (err) {
+                console.error('err', err);
+                process.exit(1);
+            }
+
+            const fields = data
+                .split(/\r?\n/)
+                .filter((line) => !!line)
+                .map((line) => ({
+                    label: line.split('=')[0],
+                    value: line.split('=')[1],
+                    type: 'concealed',
+                })) as unknown as FullItemAllOfFields[];
+
+            let existingItem;
+            if (name) {
```

Comment:
Waarom checkt ge hier op name, als ge die var niet nodig hebt in het if statement?